scripts.js

```diff
@@ -0,0 +1,10 @@
+function drawTree(i) {
+	var i = 5;
```

Comment:
??? you have an iterator i in loop, also you have an argument i in arguments, why you add this variable? also use your argument from function